docs/source/deployguide/ca-config.md

```diff
@@ -0,0 +1,417 @@
+
+- **`tls.certfile`**: Similar to the `tls.certfile`, provide the name and location of the generated TLS private key for this CA. For example: `/msp/keystore/87bf5eff47d33b13d7aee81032b0e8e1e0ffc7a6571400493a7c_sk`. If you are using an HSM or if this is a root TLS CA, this field will be blank.
+
+**Note:** In order to use the Fabric [Service Discovery](../discovery-overview.html) and [Private Data](../private-data-arch.html) features, `server-side TLS` needs to be enabled.
+
+If server-side TLS is sufficient for your needs, you are done with this section. If you need to use mutual TLS in your network then you need to configure the following two additional fields. (Mutual TLS is disabled by default.)
+
+- **`tls.clientauth.type`**: If the server additionally needs to authenticate the identity of the _client_, then **mutual TLS** (mTLS) is required. When mTLS is configured, the client is required to send its certificate during a TLS handshake. To configure your CA for mTLS, set the `clientauth.type` to `RequireAndVerifyClientCert`.
+
+- **`tls.clientauth.certfiles`**: For mTLS only, provide the PEM-encoded list of root certificate authorities that the server uses when verifying client certificates.
+
+## cafiles
+
+```
+cafiles:
+```
+
+As mentioned in the topic on [Planning for a CA](ca-deploy-topology.html), the `cafiles` parameter can be used to configure a dual-headed CA -- a single CA that under the covers includes both a root CA and a TLS CA. This usage pattern can be for convenience, allowing each CA to maintain its own configuration but still share the same back-end user database. In the `cafiles` parameter, enter the path to the `fabric-ca-server-config.yaml` of the second CA server, for example the TLS CA. The configuration of the secondary CA can contain all of the same elements as are found in the primary CA server config file except `port`, `ca.name`, and `tls` sections.
+
+If this is not a desired configuration for your CA, you can leave the value of this parameter blank.
+
+## intermediate CA
+```
+intermediate:
+  parentserver:
+    url:
+    caname:
+
+  enrollment:
+    hosts:
+    profile:
+    label:
+
+  tls:
+    certfiles:
+    client:
+      certfile:
+      keyfile:
+```
+
+Intermediate CAs are not required, but to reduce the risk of your root CA becoming compromised, you may want to include one or more intermediate CAs in your network.
+
+**Important:** Before setting up an intermediate CA, you need to verify that the `csr.ca.pathlength` parameter in the parent CA, is at least `1`. When set to `1`, the root CA can issue intermediate CA certificates, but these intermediate CAs may not in turn enroll other intermediate CAs. If you want your intermediate CA to be able to enroll other intermediate CAs, the root ca `pathlength` needs to be set to `1`.
+
```

Comment:
“If you want your intermediate CA to be able to enroll other intermediate CAs, the root ca pathlength needs to be set to 1”.

This contradicts the sentence above. "When set to `1`, the root CA can issue intermediate CA certificates, but these intermediate CAs may not in turn enroll other intermediate CAs."

Did you mean 2 or above?

Reply:
Indeed.